core/controllers/feedback.py

```diff
@@ -35,7 +35,7 @@ class ThreadListHandler(base.BaseHandler):
     def get(self, exploration_id):
         self.values.update({
             'threads': [t.to_dict() for t in feedback_services.get_all_threads(
-                exploration_id, False)]})
+                'exploration', exploration_id, False)]})
```

Comment:
Should be using a named constant instead of 'exploration', here and elsewhere.

Reply:
Done